javascript/modules/client/src/client.ts

```diff
@@ -327,7 +327,7 @@ export class Client extends ClientCore implements IClient {
 
   //// PRIVATE METHODS METADATA
 
-  private _getMetadata(daoAddressOrEns: string): Promise<DaoMetadata> {
+  private async _getMetadata(daoAddressOrEns: string): Promise<DaoMetadata> {
```

Comment:
Why async without an `await`?

Reply:
Forgot to delete it after modifying the contents of the function